batcher/aligned-sdk/src/communication/batch.rs

```diff
@@ -0,0 +1,85 @@
+use log::debug;
+
+use crate::{
+    core::{
+        errors,
+        types::{
+            AlignedVerificationData, BatchInclusionData, Chain, VerificationCommitmentBatch,
+            VerificationDataCommitment,
+        },
+    },
+    sdk::verify_proof_onchain,
+};
+
+const AWAIT_BATCH_VERIFICATION_RETRIES: u64 = 10;
+const AWAIT_BATCH_VERIFICATION_TIMEOUT: u64 = 10;
```

Comment:
this is not the timeout but the time between retries

Reply:
Solved in [617be52](https://github.com/yetanotherco/aligned_layer/pull/583/commits/617be520af0af074a57ce3d3089384b97ebb5293)